Marlin/src/lcd/extui/lib/mks_ui/wifiSerial.cpp

```diff
@@ -89,7 +89,11 @@ void WifiSerial::begin(uint32 baud, uint8_t config) {
                            txi->gpio_device, txi->gpio_bit,
                            config);
   usart_set_baud_rate(this->usart_device, USART_USE_PCLK, baud);
-  usart_enable(this->usart_device);
+  //usart_enable(this->usart_device);
```

Comment:
delete?

Reply:
I think this is the type of the code that must stay there, even commented... because seems it's not clear why enable is commented...